packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -346,6 +346,13 @@ Future<String> _buildAotSnapshot(
       printError('Compiler terminated unexpectedly.');
       return null;
     }
+
+    if (strongMode) {
+      genSnapshotCmd.addAll(<String>[
+        '--reify-generic-functions',
+        '--strong',
```

Comment:
does it also need --limit-ints-to-64-bits ?

Reply:
Those are on by default now AFAIK.

Reply:
https://github.com/dart-lang/sdk/blob/master/runtime/vm/flag_list.h#L103